pandas/io/stata.py

```diff
@@ -1341,12 +1341,14 @@ def _null_terminate(self, s):
                 return s
 
     def _read_value_labels(self):
-        if self.format_version <= 108:
-            # Value labels are not supported in version 108 and earlier.
-            return
         if self._value_labels_read:
             # Don't read twice
             return
+        if self.format_version <= 108:
+            # Value labels are not supported in version 108 and earlier.
+            self._value_labels_read = True
+            self.value_label_dict = dict()
```

Comment:
Was there a preference stated recently to use `{}` rather than `dict()`?

Reply:
yeah can use ``{}``

Reply:
`dict()` is internally consistent with the function.